src/act.core.web/Services/NodeFactory.cs

```diff
@@ -252,6 +252,12 @@ public async Task<NodeSearchResult[]> GetAssignedToBuildSpec(long buildSpecId)
 
                     q = q.OrderBy(p => p.BuildSpecification.Name).ThenBy(p => p.Fqdn);
                     break;
+                case NodeSearchTypeConstant.RemedyGroupEmail:
+                    if (!string.IsNullOrWhiteSpace(query))
+                        q = q.Where(p => EF.Functions.Like(p.RemedyGroupEmailList, $"{query}%"));
+
+                    q = q.OrderBy(p => p.RemedyGroupEmailList);
```

Comment:
I know this is existing code, but I have no idea what q is here, we should always use descriptive variable names.

Reply:
changed to node